camshare/camshare/PhotoAlbumViewController.swift

```diff
@@ -49,8 +55,20 @@ extension PhotoAlbumViewController: UICollectionViewDataSource {
         //swiftlint:disable all
         let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "PhotoCell", for: indexPath) as! PhotoCell
         //swiftlint:enable all
-        let image = images[indexPath.item]
-        cell.imageView.image = image
-        return cell
-    }
+        let tapGestureRecognizer = UITapGestureRecognizer(target: self,
```

Comment:
Every collection view cell and tableview sell has a click gesture on it, you do not need to do this yourself. https://developer.apple.com/documentation/uikit/uitableviewdelegate/handling_row_selection_in_a_table_view

Reply:
Oky cool Thanks